translations/messages.fr.yaml

```diff
@@ -14,34 +14,57 @@ action:
   selectAll: Tout sélectionner
   save: Enregistrer
   submit: Valider
+asset:
+  comments: Commentaires
+  contact: Qui contacter ?
+  hasMobileRadio: Présence d'un mobile radio ?
+  hasFirstAidKit: Présence d'un lot de secours ?
+  licensePlate: Plaque d'immatriculation
+  lightVehicle: Véhicule léger
+  name: Identifiant
+  parkingLocation: Lieu de stationnement
+  planning:
+    hasMobileRadio: Mobile radio ?
+    hasFirstAidKit: Kit de premiers soins ?
+    contact: Contact
+    seatingCapacity: Places
+    licensePlate: Immatriculation
+  seatingCapacity: Combien de places ?
+  type: Type
 calendar:
   displayedPeriod: Période affichée
   hours: heures
   period: du %from% au %to%
   schedule: Planning
   showAvailableOnly: Afficher seulement si disponible sur la période
   sortByDayAvailabilities: Trier par nombre de disponibilités sur le jour
+  to: à
   week:
     current: Semaine actuelle
     next: Semaine prochaine
     startMonday: Semaine du lundi %week%
 common:
   actions: Actions
   availabilities: Disponibilités
+  backToList: Retourner à la liste
   informations: Informations
   manager: Cadre
   name: Nom
+  number: Nombre
+  no: Non
   occupation: Profession
   phoneNumberShort: Tél
+  type: Type
   yes: Oui
-  no: Non
-  back_to_list: Retourner à la liste
 message:
+  deleteAssetWarning: "Vous êtes sur le point de supprimer le véhicule : <span id='to-delete-name'></span> et toutes ses disponibilités."
+  deleteSlotWarning: "Vous êtes sur le point de modifier la disponibilité de <strong id='nb-assets-users'></strong>"
```

Comment:
it can be dangerous to put some important html in the translations. Maybe you could change the sentence and keep the html tag in the view : `deleteSlotWarning: "Vous êtes sur le point de modifier la disponibilité de: "` and ` {{ 'message.deleteSlotWarning' | trans }} <strong id='nb-assets-users'></strong>`